impress-org/givewp
# Only display login form in Donor Dashboard when there are forms that allow login

Resolves #6011

## Description



After a good deal of user feedback, we determined that it does not make sense to display the login option on the Donor Dashboard authentication model if there are no forms that allow logging in.

To be honest, given that the Donor Dashbaord is for all forms on a site, if there are websites that have more than a single donation form with varying options set for allowing donors to login or register, this still will remain unclear and I don’t think there is anyway to make it clear. I think this PR is still valuable as I suspect that most sites have a single form or if they have multiple, they probably use the same options.

## Affects



- Donor Dashboard authentication model

## Visuals



- Will add.

## Testing Instructions



- Build assets (`npm run build`).
- Make a few donation forms with the "Register" setting to "None" — in this case the login should not display in the donor dashboard.
- Change one of the forms to have a non-"None" option — in this case the login should display in the donor dashboard.

## Pre-review Checklist
